### ISSUE TITLE
Need to specify type to `rm`

### ISSUE THREAD

**Issue by liftedkilt** [danielpigott/cloudflare-cli#28]
when issuing a `cfcli rm`, there's no way to specify the record type. So If I have the following two records:

```
CNAME,blah.example.com,blah.com,Auto,false
TXT,blah.example.com,blah.com,Auto,false

```
a `cfcli rm blah blah.com` will remove both records.

**Comment by danielpigott** [comment]
This should be relatively straight forward to fix, I will include it in the next update

**Comment by adenau** [comment]
I would be a bit afraid of this, as it could create unintended side effects. I agree with not having to specify a type, but maybe the command should stop when two(or more) records would be deleted. Something like

"Cowardly refusing to execute ambiguous command. More than one entry would be deleted."

**Comment by liftedkilt** [comment]
type could be an optional flag. I agree with @adenau it should fail if there are multiple records and type isn't specified.

**Comment by danielpigott** [comment]
I have included a fix for this in 2.1.0.  It will now match the type if passed in. It will still delete multiple records. I will look at how to deal with this in the next release, as I have to do some work on the error messages in general

**Comment by danielpigott** [comment]
I have now added the --query parameter that you can you to target records of a particular value or type (see readme). I'm going to resolve this issue, but please feel free to open a new one if there are any further issues around this
